src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -71,7 +71,7 @@ foam.CLASS({
       name: 'subfooterLink',
       code: function() {
         this.stack.push({
-          class: 'foam.nanos.auth.resetPassword.ForgotPasswordView'
+          class: 'foam.nanos.auth.ChangePasswordView'
```

Comment:
So based off of Patrick comments which I agree with this should pass in the SendPassword model here.

and I do think there are more calls to the passwordView which will need to updated. SORRY ... but also thank you!!